easybuild/easyconfigs/x/Xerces-C++/Xerces-C++-3.2.2-GCCcore-8.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
Is there any reason to change from `CMakeMake` as it is already used in `Xerces-C++-3.2.2-GCCcore-8.2.0.eb`?

Reply:
Revert to CMakeMake